juniper_official/vpn/evpn/evpn-check-mac-count-netconf.rule

```diff
@@ -0,0 +1,126 @@
+/*
+ *   This rule checks health of MAC count in evpn mac table and notify in case any of the health monitored field crosses threshold
+ *
+ * Requirement 455 - https://paragon-automation.atlassian.net/browse/REQ-455
+ *
+ */
+healthbot {
+    topic evpn {
+        description "This rule checks health of MAC count in evpn mac table and notify in case any of the health monitored field crosses threshold";
+        synopsis "MAC count check in evpn";
+        rule evpn-check-mac-count-netconf {
+            keys instance-name;
+            synopsis "MAC count in evpn mac table analyzer";
+            description "This rule checks health of MAC count in ethernet switching table and notify in case any of the health monitored field crosses threshold";
+            sensor evpn-mac-count {
+                synopsis "mac count netconf based iAgent sensor definition";
+                description "iAgent sensor defined in yml file using netconf rpc/cli command and defined fields using pyez tables&views to collect data from network device";
+                iAgent {
+                    file evpn-mac-count.yml;
+                    table MacCountTable;
+                    frequency 180s;
+                }
+            }
+            /*
+             * Fields defined using sensor path. Map the longer sensor names
+             * to the shorter field names used in the rules.
+             */
+            field mac-count {
+                sensor evpn-mac-count {
+                    path mac-count;
+                }
+                type integer;
+                description "This filed is for mac count";
+            }
+            field instance-name {
+                sensor evpn-mac-count {
+                    where "instance-name =~ /{{instance-name}}/";
+                    path instance-name;
+                }
+                type string;
+                description "This filed is for rtb name in mac table";
+            }
+            field threshold {
+                constant {
+                    value "{{static-threshold}}";
+                }
+                type integer;
+            }
+            field learn-vlan {
+                sensor evpn-mac-count {
+                    path learn-vlan;
+                }
+                type string;
+                description "VLAN ID";
+            }
+            /*
+             * Anomaly detection logic.
+             */
+            trigger mac-count {
```

Comment:
Can you enahnce trigger name to include which Mac count? Not that these will be alerts and so should make sense.
For example "down" can represent any resource down. "link-donw" or "interface-down" represents properly what is being impacted.

Reply:
This KPI is to monitor evpn mac count whether count is with in threshold are not  and also count value will be informative hence evpn-mac-cout make sense